packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -1856,4 +1856,191 @@ describe('RpcContractProvider test', () => {
+                  {
+                    "prim": "DIG",
+                    "args": [
+                      {
+                        "int": "2"
+                      }
+                    ]
+                  },
+                  {
+                    "prim": "UPDATE"
+                  },
+                  {
+                    "prim": "NIL",
+                    "args": [
+                      {
+                        "prim": "operation"
+                      }
+                    ]
+                  },
+                  {
+                    "prim": "PAIR"
+                  }
+                ]
+              ]
+            }
+          ],
+          "storage": [
+            {
+              "prim": "Elt",
+              "args": [
+                {
+                  "string": "tz1QYD1zbK2gTUu1YWX8m7hPcKNkuXoxPo73"
+                },
+                {
+                  "prim": "Pair",
+                  "args": [
+                    {
+                      "string": "KT19mzgsjrR2Er4rm4vuDqAcMfBF5DBMs2uq"
+                    },
+                    {
+                      "bytes": "0505080a0000001601f37d4eddfff4e08fb1f19895ac9c83bc12d2b36800"
+                    },
+                    {
+                      "int": "2"
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "prim": "Elt",
+              "args": [
+                {
+                  "string": "tz1cor8JEddCMvLFpWBK1EcNFDU3QgaSwvc1"
+                },
+                {
+                  "prim": "Pair",
+                  "args": [
+                    {
+                      "string": "KT19mzgsjrR2Er4rm4vuDqAcMfBF5DBMs2uq"
+                    },
+                    {
+                      "bytes": "0505080a0000001601f37d4eddfff4e08fb1f19895ac9c83bc12d2b36800"
+                    },
+                    {
+                      "int": "10000"
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "prim": "Elt",
+              "args": [
+                {
+                  "string": "tz1h5GajcQWq4ybaWuwSiYrR5PvmUxndm8T8"
+                },
+                {
+                  "prim": "Pair",
+                  "args": [
+                    {
+                      "string": "KT19mzgsjrR2Er4rm4vuDqAcMfBF5DBMs2uq"
+                    },
+                    {
+                      "bytes": "050505030b"
+                    },
+                    {
+                      "int": "1000000"
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      })
+      const rpcContract = await rpcContractProvider.at('KT19mzgsjrR2Er4rm4vuDqAcMfBF5DBMs2uq');
+      const storage = await rpcContract.storage() as any;
+      expect(rpcContract).toBeDefined();
+
+      const keyList = storage.keyMap;
+      expect(keyList.size).toEqual(3);
```

Comment:
It would be interesting to see how the ticket is represented in the storage as well.

This test is great, I think we should add more tests like that for map having different key and value:
- map having a map as key or value
- map having a bigmap as key or value
- map having [any other more complex type] as key or value